proto/scheduler_controller.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+message Worker {
+  string name = 1;
+}
+
+enum RegisterStatus {
+  UNKNOWN = 0;
+  OK = 1;
+  REJECTED = 2;
+}
+
+service Controller {
+  rpc Register(Worker) returns (RegisterStatus );
+}
```

Comment:
I don't think this service is needed.

Reply:
We will bulk update the proto with everything needed